config/runtime.exs

```diff
@@ -18,7 +18,7 @@ if config_env() == :prod do
       # See the documentation on https://hexdocs.pm/plug_cowboy/Plug.Cowboy.html
       # for details about using IPv6 vs IPv4 and loopback vs public addresses.
       ip: {0, 0, 0, 0, 0, 0, 0, 0},
-      port: String.to_integer(System.get_env("PORT") || "4000")
+      port: 8080
```

Comment:
Is this the same default port as Livebook? If so, we should change it to avoid conflicts. You can pick whatever you want. :)

Reply:
`8090` then :)